File: AbyssalCreatures/src/main/java/com/offz/spigot/abyssalcreatures/Mobs/Hostile/Silkfang.java
package com.offz.spigot.abyssalcreatures.Mobs.Hostile;

import com.offz.spigot.mobzy.mobs.behaviours.HitBehaviour;
import com.offz.spigot.mobzy.mobs.types.HostileMob;
import com.offz.spigot.mobzy.pathfinders.PathfinderGoalWalkingAnimation;
import net.minecraft.server.v1_13_R2.World;

public class Silkfang extends HostileMob implements HitBehaviour {
    public Silkfang(World world) {
        super(world, "Silkfang");
        this.setSize(2F, 2F);
    }

    @Override
    public void createPathfinders() {
        super.createPathfinders();
        goalSelector.a(0, new PathfinderGoalWalkingAnimation(this, getStaticBuilder().getModelID()));
    }
}

File: AbyssalCreatures/src/main/java/com/offz/spigot/abyssalcreatures/mobs/hostile/Silkfang.kt
package com.offz.spigot.abyssalcreatures.mobs.hostile

import com.offz.spigot.mobzy.mobs.behaviours.HitBehaviour
import com.offz.spigot.mobzy.mobs.types.HostileMob
import net.minecraft.server.v1_15_R1.PathfinderGoalLeapAtTarget
import net.minecraft.server.v1_15_R1.World

class Silkfang(world: World?) : HostileMob(world, "Silkfang"), HitBehaviour {
    override fun createPathfinders() {
        super.createPathfinders()
        addPathfinderGoal(1, PathfinderGoalLeapAtTarget(this, 0.3f))
    }
}